lonestardist/tc/tc_cuda.cu

```diff
@@ -6,13 +6,12 @@
 
 void kernel_sizing(CSRGraph &, dim3 &, dim3 &);
 #define TB_SIZE 256
-#include "kernels/reduce.cuh"
+#include "moderngpu/kernel_reduce.hxx"
 #include "tc_cuda.cuh"
-#include "kernels/segmentedsort.cuh"
-#include "moderngpu.cuh"
-#include "util/mgpucontext.h"
+#include "moderngpu/kernel_segsort.hxx"
 #include <cuda_profiler_api.h>
-mgpu::ContextPtr mgc;
+using namespace mgpu;
```

Comment:
I generally like explicit namespaces (i.e., avoiding `using namespace XYZ`) as it is easier to see where symbols come from.

In this case, we still explicitly refer to `cub` like in `cub::BlockReduce` below. Is there a reason to use the `mpgu` namespace but not `cub`?

Reply:
Sure, we can use the explicit namespace. I will fix the code.